Sming/third-party/.patches/esp-open-lwip.patch

```diff
@@ -130,19 +130,64 @@ index ff03b30..fde6567 100644
  #define LWIP_PLATFORM_DIAG(x)
  #define LWIP_PLATFORM_ASSERT(x)
 diff --git a/include/lwipopts.h b/include/lwipopts.h
-index eaa8dd6..dbacd90 100644
+index eaa8dd6..6568657 100644
 --- a/include/lwipopts.h
 +++ b/include/lwipopts.h
-@@ -953,7 +953,7 @@
+@@ -403,7 +403,7 @@
+  * PBUF_POOL_SIZE: the number of buffers in the pbuf pool. 
+  */
+ #ifndef PBUF_POOL_SIZE
+-#define PBUF_POOL_SIZE                  10
++#define PBUF_POOL_SIZE                  8
+ #endif
+ 
+ /*
+@@ -422,7 +422,7 @@
+  * ARP_TABLE_SIZE: Number of active MAC-IP address pairs cached.
+  */
+ #ifndef ARP_TABLE_SIZE
+-#define ARP_TABLE_SIZE                  10
++#define ARP_TABLE_SIZE                  8
+ #endif
+ 
+ /**
+@@ -447,7 +447,7 @@
+  * Also notice that this slows down input processing of every IP packet!
+  */
+ #ifndef ETHARP_TRUST_IP_MAC
+-#define ETHARP_TRUST_IP_MAC             1
++#define ETHARP_TRUST_IP_MAC             0
```

Comment:
For applications that connect with lots of new clients this should be 1. For applications where you stay connected to the cloud and occasionally to a mobile device in the house (iot) i think this should be 0 because the macs will already be in the arp cache for your router + other devices in the house